include/zephyr/usb/class/usbd_uvc.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2025 tinyVision.ai Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief USB Video Class (UVC) public header
+ *
+ * Header exposes API for registering Video devices to UVC instances.
+ */
+
+#ifndef ZEPHYR_INCLUDE_USB_CLASS_USBD_UVC_H
+#define ZEPHYR_INCLUDE_USB_CLASS_USBD_UVC_H
+
+#include <zephyr/kernel.h>
+
+void uvc_set_video_dev(const struct device *const dev, const struct device *const video_dev);
```

Comment:
an open question here but in the `@file` comment you're using the "register" terminology ("Header exposes API for registering Video devices to UVC instances") -- should the API be named `uvc_register_video_dev`?

Reply:
Good point! Though:

- Multiple call to `uvc_register_video_dev()` would make it look like several video devices would be registered but this not what it does...

- Multiple calls to `uvc_set_video_dev()` would make it visible that it instead overwrites the previous assignment with the new one.

Maybe that means I need to update the other side (the `@file`).